pkg/products/threescale/reconciler.go

```diff
@@ -864,6 +864,10 @@ func (r *Reconciler) resyncRoutes(ctx context.Context, client k8sclient.Client)
 		k8sclient.MatchingLabels(map[string]string{"deploymentConfig": "system-sidekiq"}),
 	}
 	err := client.List(ctx, pods, listOpts...)
+	if err != nil {
+		r.log.Error("Error getting list of pods", err)
```

Comment:
```suggestion
		r.log.Error("error getting list of pods", err)
```

It seems error logs in other files will need editing as well